.travis.yml

```diff
@@ -48,8 +48,12 @@ install:
     $PIP install --upgrade pip
     $PIP install --upgrade wheel
     $PIP install --upgrade setuptools
+    echo "Travis build stage: $TRAVIS_BUILD_STAGE_NAME"
     # install dependencies yt
-    if [[ $TRAVIS_BUILD_STAGE_NAME != "Lint" ]]; then
+    if [[ ${TRAVIS_BUILD_STAGE_NAME} == "lint" ]]; then
+      export TRAVIS_BUILD_STAGE_NAME="Lint"
+    fi
+    if [[ ${TRAVIS_BUILD_STAGE_NAME} != "Lint" ]]; then
```

Comment:
Here and below, what is the reason to use `Lint` instead of `lint`?

Reply:
I suppose it could be either, but basically what seemed to be happening is that while previously Travis had been capitalizing the stage names (i.e., `lint` became `Lint`) for some reason it no longer was in our jobs.  And, because OS X runs bash 3.2, we couldn't use the `${VAR^^}` or `${VAR,,}` construction to change the case as a whole.

So, this is in there in case it switches back.  It could use either one, but we figured that the least invasive one would be to assume the old behavior would return some day.